storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -8127,6 +8175,12 @@ int main(int argc, char **argv) {
     exit(EXIT_FAILURE);
   }
 
+  if (xtrabackup_parallel_stream_set && xtrabackup_stream_dir == NULL &&
+      strcmp(xtrabackup_target_dir, "./xtrabackup_backupfiles/") == 0) {
```

Comment:
Because that is the default value for xtrabackup_target_dir (it inherits from xtrabackup_real_target_dir). Here we are erroring out if neither --stream-dir nor --target-dir has been specified.

--stream-dir I have added to be compatible with xbcloud and xbstream, so all tree tools use the same parameters
--target-dir I left as an option for backward compatibility.